draft-fossati-tls-exported-attestation.md

```diff
@@ -72,38 +86,119 @@ There is a growing need to demonstrate to a remote party that cryptographic keys
+
+
+## Negotiation of CMWAttestationExtension
+
+Clients and servers use the TLS flags extension defined in {{I-D.ietf-tls-tlsflags}} to indicate support for the functionality defined in this document. We refer to the previously defined "cmw_attestation" extension,
+and the corresponding flag is called the "CMW_Attestation" flag.
+
+The "CMW_Attestation" flag proposed by the client in the ClientHello MUST be acknowledged in the EncryptedExtensions if the server also  supports the functionality defined in this document and is configured to use it.
+
+If the "CMW_Attestation" flag is not set, servers ignore any of the functionality specified in this document, and attestation evidence cannot be conveyed using "Exported TLS Authenticators".
+
+## Usage in Post-Handshake Authentication
+
+The `cmw_attestation` extension is designed to be used exclusively in post-handshake authentication as defined in {{RFC9261}}. It allows attestation evidence to be transmitted in the authenticator (Certificate) message only in response to an authenticator request. This ensures that attestation data is provided on demand rather than being included in the initial TLS handshake.
+
+To maintain a cryptographic binding between the attestation evidence and the authentication request, the `cmw_attestation` extension MUST be associated with the certificate_request_context of the corresponding CertificateRequest or ClientCertificateRequest message. This binding ensures that:
+
+- The attestation evidence is specific to the authentication event and cannot be replayed across different TLS sessions.
+- The attestation remains tied to the cryptographic context of the TLS session.
+
+## Ensuring Compatibility with X.509 Certificate Validation
+
+The `cmw_attestation` extension does not modify or replace X.509 certificate validation mechanisms. It serves as an additional source of authentication data rather than altering the trust model of PKI-based authentication. Specifically:
+
+- Certificate validation (e.g., signature verification, revocation checks) MUST still be performed according to TLS {{!RFC8446}} and PKIX {{!RFC5280}}.
+- The attestation evidence carried in `cmw_attestation` MUST NOT be used as a substitute for X.509 certificate validation but can be used alongside standard certificate validation for additional security assurances.
+- Implementations MAY reject connections where the certificate is valid but the attestation evidence is missing or does not meet security policy.
+
+## Applicability to Client and Server Authentication
+
+The `cmw_attestation` extension is applicable to both client and server authentication in Exported Authenticator-based post-handshake authentication.
+
+### Client Authentication
+For client authentication, the server acts as the relying party and requests attestation from the client. The client may respond with either:
+
+- Evidence (Passport Model):
```

Comment:
`Background Check Model` here, and `Passport Model` on line 161.

Same for lines 170 and 175.

Reply:
Good catch, fixed.